superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/plugin/controlPanel.ts

```diff
@@ -196,6 +196,46 @@ const config: ControlPanelConfig = {
         ],
       ],
     },
+    {
+      label: t('IP dashboard link configuration'),
+      expanded: true,
+      tabOverride: 'data',
+      controlSetRows: [
+        [
+          {
+            name: 'ipDashBoardBaseUrl',
+            config: {
+              type: 'TextControl',
+              label: t('Base url for supersert'),
+              renderTrigger: true,
+              description: t('the url for superset you wish to link to. i.e superset.com'),
+            },
+          }
+        ],
+        [
+          {
+            name: 'ipDashboardId',
+            config: {
+              type: 'TextControl',
+              label: t('Ip dasboard id'),
```

Comment:
Since this is the label should we go with: 'IP dashboard ID' or 'IP dashboard Id', whatever the team's preferred way of display id is.